src/scrubber.ts

```diff
@@ -6,12 +6,15 @@ export class Scrubber {
 
   constructor (private cfg: ScrubberConfig, additionalScrubbersImpl?: ScrubbersImpl) {
     this.scrubbers = { ...defaultScrubbers, ...additionalScrubbersImpl }
-    cfg = this.expandCfg(cfg)
+    this.cfg = this.expandCfg(cfg)
     this.checkIfScrubbersExistAndRaise(cfg, this.scrubbers)
   }
 
   scrub<T extends any[]> (...data: T): T {
-    return this.applyScrubbers(data)
+    const result = this.applyScrubbers(data)
+
+    if (data.length === 1) return result[0]
```

Comment:
Oh no, do you know a better way to do this?

The issue is that when we remove `scrubSingle` and made `scrub()` support both single object and an array of objects, for single objects it returns an array, which I think is unexpected.